<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: white; 
    color: #333333;          /* Dark text for all labels and header */
}

QLabel {
    color: #333333;          /* Ensures all labels (Name, Age, Header) are dark gray */
}


QLineEdit, QDateEdit, QDateTimeEdit {
    color: #000000;            /* Text typed in is black */
    background-color: white;   /* &lt;-- FORCES WHITE BACKGROUND HERE */
    border: 1px solid #CCCCCC;
    padding: 5px;
    border-radius: 3px;
}

QDateTimeEdit::up-button, QDateTimeEdit::down-button {
    /* Set a minimum width and height for the buttons */
	color: #000000;  
    min-width: 15px; 
    min-height: 15px;
    
}


/* 3. BUTTONS (We target the QDialogButtonBox child buttons) */
QPushButton {
    border-radius: 5px; 
    padding: 5px 10px;
    font-weight: bold;
    min-width: 70px;
    color: black; /* Ensure button text is black */
}

/* Style the SAVE button (Assuming it's the default 'Ok' or you named it 'Save') */
QPushButton[text=&quot;Save&quot;], QPushButton[text=&quot;OK&quot;] { 
    background-color: #B0E0E6; /* Light Blue/Pastel Accent */
    border: none;
}

/* Style the CANCEL button to be neutral */
QPushButton[text=&quot;Cancel&quot;] { 
    background-color: #F0F0F0; 
    border: 1px solid #CCCCCC;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>521</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add New Patient</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter Patient's Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Age :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_age">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter Patient's Age</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_phone">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter 10-digit Contact Number</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registration Date :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Billed Amount :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_billedamt">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter Outstanding Amount in Rupees</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Outstanding Amount :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_outstandingamt">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter Outstanding Amount in Rupees</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
